src/common/utils/invoiceUtils.js

```diff
@@ -12,11 +13,7 @@ export const getInvoiceStatusLabel = ({ status = '' }) => (
 
 export const formatAmount = (amount = '') => `$${amount.toLocaleString('en')}`;
 
-export const formatDate = (dateString) => {
-  const date = moment.utc(dateString || '');
-
-  return date.isValid() ? date.format(DATE_FORMAT) : '';
-};
+export const formatDate = (dateString) => <FolioFormattedDate value={dateString} />;
```

Comment:
Do we need to have this function or use `<FolioFormattedDate />` directly in components

Reply:
This is a level of abstraction that saved our time and money in this case